ansible/group_vars/all.yml

```diff
@@ -85,6 +84,25 @@ virtual_machines:
               - "{{ lookup('file', gcp_ssh_key_file + '.pub', errors='ignore') }}"
     container_engine: podman
 
+  rhcos-arm64:
+    project: rhcos-cloud
+    arch: arm64
+    machine_type: t2a-standard-2
+    images:
+      - "{{ lookup('ansible.builtin.pipe', 'scripts/fetch_ocp_rhcos_bootimage.sh 4.19 aarch64.images.gcp.name') }}"
+      - "{{ lookup('ansible.builtin.pipe', 'scripts/fetch_ocp_rhcos_bootimage.sh 4.18 aarch64.images.gcp.name') }}"
+      - "{{ lookup('ansible.builtin.pipe', 'scripts/fetch_ocp_rhcos_bootimage.sh 4.15 aarch64.images.gcp.name') }}"
```

Comment:
It's a bit odd that you are defining 4.19, 4.18 and 4.15 here but ansible named the vms 4.18, 4.15 and 9.6:
```
ci-rhcos-415-92-202-7910dc53 : ok=26   changed=16   unreachable=0    failed=0    skipped=4    rescued=0    ignored=1   
ci-rhcos-418-94-202-cc26488c : ok=26   changed=15   unreachable=0    failed=0    skipped=4    rescued=0    ignored=1   
ci-rhcos-9-6-202505-d5dd64e7 : ok=26   changed=15   unreachable=0    failed=0    skipped=4    rescued=0    ignored=1   
```
Any idea why this may be happening? Is 4.19 simply using the RHEL version?

Reply:
This is because 4.19 is pre-release and the RHCOS image it isn't stamped with an OCP version yet.